enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1408 @@
+from being drained, or it may be too expensive to make a workload fully HA). When confronted with 
+problematic workloads, the current, fully self-managed, OpenShift update process can appear to the end-user
+to be unreliable or slow. This is because the self-managed update process takes on the end-to-end responsibility
+of updating the control-plane and worker-nodes. Given the automated and somewhat opaque nature of this
+update, it is reasonable for users to expect that the process is hands-off and will complete in a timely 
+manner regardless of their workloads.  
+
+When this expectation is violated because of problematic user workloads, the update process is 
+often called into question. For example, if an update appears stalled after 12 hours, a
+user is likely to have a poor perception of the platform and open a support ticket before
+successfully diagnosing an underlying undrainable workload.
+
+By separating control-plane and worker-node updates into two distinct phases for an operator to consider,
+we can more clearly communicate (1) the reliability and speeed of OpenShift control-plane updates and
+(2) the shared responsibility, along with the end user, of successfully updating worker-nodes. 
+
+As an analogy, when you are late to work because of delays in a subway system, you blame the subway system.
+They own the infrastructure and schedules and have every reason to provide reliable and predictable transport.
+If, instead, you are late to work because you step into a fully automated car that gets stuck in traffic, you blame the
+traffic. The fully self-managed update process suggests to the end user that it is a subway -- subtly insulating
+them from the fact that they might well hit traffic (problematic user workloads). Separating the update journey into
+two parts - a subway portion (the control-plane) and a self-driving car portion (worker-nodes), we can quickly build the
+user's intuition about their responsibilities in the latter part of the journey. For example, leaving earlier to
+avoid traffic or staying at a hotel after the subway to optimize their departure for the car ride.
+
+##### Node Update Separation to Improve Risk Mitigation Strategies
+With any cluster update, there is risk --  software is changing and even subtle differences in behavior can cause
+issues given an unlucky combination of factors. Teams responsible for cluster operations are familiar with these
+risks and owe it to their stakeholders to minimize them where possible.
+
+The current, fully self-managed, update process makes one obvious risk mitigation strategy
+a relatively advanced strategy to employ: only updating the control-plane and leaving worker-nodes as-is.
+It is possible by pausing machine config pools, but this is certainly not an intuitive step for users. Farther back 
+in OpenShift 4's history, the strategy was not even safe to perform since it could lead to worker-node 
+certificates to expiring. 

```

Comment:
Not a native speaker, but:
"since it could lead to worker-node  certificates to expiring. "   to ... to ... sounds strange